src/components/content/gender/DisplayGender.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, useEffect } from "react";
+import createDonutChart from "./createDonutChart";
+import { IoMdFemale, IoMdMale } from "react-icons/io";
+
+interface DisplayGenderProps {
+  gender: string;
+  genderPercentage: number;
+}
+
+const DisplayGender: FC<DisplayGenderProps> = (props) => {
+  const showGenderIcon = (gender: string) => {
+    if (gender === "female") {
+      return <IoMdFemale className="gender-icon" />;
+    } else if (gender === "male") {
```

Comment:
install prettier(esbenp.prettier-vscode) and eslint(dbaeumer.vscode-eslint) VSC extensions to avoid `else` after `return` situations